src/components/data-management/AnalysisModal.jsx

```diff
@@ -32,9 +33,12 @@ const NewExperimentModal = (props) => {
   const [isWorking, setIsWorking] = useState(false);
 
   useEffect(() => {
-    setExperimentsList(
-      activeProject?.experiments?.map((experimentId) => experiments[experimentId]) || [],
-    );
+    if (!activeProject?.experiments?.length > 0) return;
+
+    const updatedList = activeProject.experiments.map((experimentId) => experiments[experimentId])
```

Comment:
I would place a newline before `.map` so it and `.filter` are lined up

Reply:
:+1: